### DISCUSSION TITLE
Optimising queries for data descriptors, for example "spontaneous activity"

### DISCUSSION THREAD

**Discussion by duutabib** [dandi/helpdesk#86] [Q&A]
Hello Everyone, 
I am looking for datasets on spontaneous activity in mice or human brains. Would you be kind to share some pointers on how to access such dandisets from the archive?

I returned two hits with the query "spontaneous activity". However, I suspect there could be more, no?

Thank you very much in advance .

**Comment by bendichter** [comment]
Honestly I don't know the answer to this. We'll need to enhance our metadata so we can filter by this sort of thing.

**Reply by satra** [reply]
@bendichter - how would you go about knowing this? anything in NWB metadata based on neural data type?

**Reply by duutabib** [reply]
@bendichter Could we define a task based on this? I will be happy to make this contribution beyond the added benefit of
getting access to my dandisets.
Thanks!

**Reply by bendichter** [reply]
@satra Right now I would rely on parsing the `NWBFile.experiment_description` field or the description of the dandiset. The ideal way to get this information would be to formally define task types. I talked to Tom and Lyuba about this at SfN and neither of them are aware of any good standards we can adopt for this. It would definitely be worth the effort to come up with a shared vocabulary for such things.

As for determining this with neurodata types, I might suspect that sessions without any trials and without any stimuli might be good candidates, but I also think that will give you a lot of false positives.

**Reply by duutabib** [reply]
@bendichter I get the part about false positives! 

Would it take more than a month to parse the `NWBFile.experiment_description`?
